.github/workflows/cypress.yaml

```diff
@@ -154,23 +154,23 @@ jobs:
               run: |
                   echo "CYPRESS_RUST_CRYPTO=1" >> "$GITHUB_ENV"
 
-            - name: Run Cypress tests
+            - name: Run Cypress tests via knapsack pro
               uses: cypress-io/github-action@ebe8b24c4428922d0f793a5c4c96853a633180e3 # v6.6.0
               with:
                   working-directory: matrix-react-sdk
-                  # The built-in Electron runner seems to grind to a halt trying to run the tests, so use chrome.
```

Comment:
We should not be removing this, we install a specific chrome version on purpose, otherwise using the default one causes issues as github actions rolls out new versions of chrome

https://github.com/cypress-io/github-action/issues/518 for context

Reply:
That's not what the comment is saying though? I'll include a comment explaining why we need to use a specific version of chrome.

Reply:
Just saw the edit; have retained original comment while adding a new comment that explains why we install chrome.

Reply:
Installing chrome won't overwrite the default system one AIUI, hence us passing a path to it instead of just "chrome" - see https://github.com/matrix-org/matrix-react-sdk/commit/3874314e13e2337727d035f14c0fa6d15fd2571c#diff-51db9e43a2bd398477ba2c88d9a38051ab300bfc1fcb91da98c6d34a9e449c46R126 - I did not update the comment

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/11863/commits/212b38b419fc2666429c3fb5c90bd2dbf543acec: wdyt? Should work based on https://docs.knapsackpro.com/cypress/reference/#command-line-arguments and https://docs.cypress.io/guides/guides/module-api#Options